CDCgov/NEDSS-Modernization
# Added Table Component consistency for Patient Phone And Email

## Description

Table Component Consistency for Patient Phone and Email

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-1716

## Steps to verify
1. Log in
2. Navigate to advanced search
3. Search for any patient
4. Navigate to patient profile and navigate to Demographics tabs and observe the Phone and Email table functioning as expected.

## Checklist before requesting a review
